### PR TITLE
Feature/1282 allow leaving consortium

### LINKED ISSUE #1282: UI: allow leaving a consortium as an owner if there are other owners
## Task Description
As title, allow leaving the a consortium if there is another owner for it

## Steps to Reproduce
Make a consortium w/ two owners, observe the second cannot leave

### PR DESCRIPTION
**Pull request by kalitopix**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our [guidelines](../CONTRIBUTING.md)
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the current behavior?** (You can also link to an open issue here)
closes #1282

* **What is the new behavior (if this is a feature change)?**
Allow leaving a consortium as an owner if there are other owners

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

**Comment by rssk** [comment]
Theres a lot of deleted file and modified content not relevant to this PR whats up with that?

File: packages/coinstac-ui/app/render/components/common/member-avatar.jsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ function MemberAvatar({
 }) {
   const { data } = useQuery(FETCH_USER_QUERY, {
     variables: { userId: id },
-    onError: (error) => { console.error({ error }); },
```
Comment (rssk) [inline packages/coinstac-ui/app/render/components/common/member-avatar.jsx]: can you add all these back and put exceptions? theyre here for a reason

File: packages/coinstac-ui/app/render/components/consortium-card.jsx [hunk 2]
```diff
@@ -1,118 +0,0 @@
-import React from 'react';
```
Comment (rssk) [inline packages/coinstac-ui/app/render/components/consortium-card.jsx:1]: whats up w this being deleted?

File: packages/coinstac-ui/app/render/components/headless/headless-edit/api-key-tab/api-key-tab.jsx [hunk 3]
```diff
@@ -36,10 +32,6 @@ function ApiKeyTab({ headlessClientData, onHeadlessClientUpdate }) {
     }
   );
 
-  function copyToClipboard() {
```
Comment (rssk) [inline packages/coinstac-ui/app/render/components/headless/headless-edit/api-key-tab/api-key-tab.jsx:39]: was this not used?

File: packages/coinstac-ui/app/render/styles/consortium-card.scss [hunk 4]
```diff
@@ -1,12 +0,0 @@
-/**
- * Consortium card.
- */
-.consortium-card {
```
Comment (rssk) [inline packages/coinstac-ui/app/render/styles/consortium-card.scss:4]: not used??

**Review by rssk — CHANGES_REQUESTED** [review]

File: packages/coinstac-ui/app/render/components/headless/headless-edit/api-key-tab/api-key-tab.jsx (hunk 3 above)
Comment (kalitopix) [reply packages/coinstac-ui/app/render/components/headless/headless-edit/api-key-tab/api-key-tab.jsx:39]: @rssk Yes

File: packages/coinstac-ui/app/render/styles/consortium-card.scss (hunk 4 above)
Comment (kalitopix) [reply packages/coinstac-ui/app/render/styles/consortium-card.scss:4]: @rssk Yes

**Pull request merged**